generators/languages/index.js

```diff
@@ -13,9 +13,39 @@ var LanguagesGenerator = generators.Base.extend({});
 
 util.inherits(LanguagesGenerator, scriptBase);
 
+var configOptions = {};
+
 module.exports = LanguagesGenerator.extend({
     constructor: function() {
         generators.Base.apply(this, arguments);
+
+        configOptions = this.options.configOptions || {};
+
+        // This makes it possible to pass `languages` by argument
+        this.argument('languagesArgument', {
```

Comment:
So the reason why I did not use the name `language` is cause it is not possible to set it in the index.js file. 

The following code: 
```javascript
        console.log("this.languagesArgument = ", this.languagesArgument);
        this.languagesArgument = ['en', 'fr'];
        console.log("this.languagesArgument = ", this.languagesArgument);
```

Prints:
```javascript
this.languagesArgument =  [ 'en' ]
this.languagesArgument =  [ [ 'en', 'fr' ] ]
```

@deepu105 any reason why? It feels like some javascript voodoo magic =)